eng/Packages.Data.props

```diff
@@ -84,7 +84,7 @@
     <PackageReference Update="System.Diagnostics.Tools" Version="4.3.0" />
     <PackageReference Update="System.Diagnostics.TraceSource" Version="4.3.0" />
     <PackageReference Update="System.Globalization" Version="4.3.0" />
-    <PackageReference Update="System.IdentityModel.Tokens.Jwt" Version="[5.4.0, 6.0.0)" />
+    <PackageReference Update="System.IdentityModel.Tokens.Jwt" Version="5.4.0" />
```

Comment:
I don't have concerns, but I would like to be sure that we have @serkantkaraca's blessing, as he would be the authority on potential impact to the track one Event Hubs usage.

Reply:
Are we aware of any breaking changes in versions >6? In general the guidance to use upper limits is only to block dependencies with known breaking changes.

Reply:
There are a few breaking changes in 6.5.0:
https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/releases/tag/6.5.0

Neither of these breaking changes is in System.IdentityModel.Tokens.Jwt assembly.

Reply:
OK if there are no breaking changes that impact our usage then I would recommend removing the upper bound.